contributor/outline.md

```diff
@@ -132,6 +132,7 @@ Outline for this section inspired by the [Trusted Committer introduction video].
     This typically makes the code more reusable, replaceable and secure.
```

Comment:
OK to include your point in the existing bullet on engineering efficiency?

```suggestion
    This typically makes the code more reusable, replaceable and secure.
    Collaboration across team boundaries saves duplicate effort and thus development cost
```

Reply:
We did not have the "shared development cost" point explicitly stated here yet. However academic research and practical insight gives this as a selling point with direct business relevance. One that could be used by possible contributors selling the approach.
Engineering Efficiency feels similar but not equal and less explicit with that regard to me. My opinion: I would keep it as it's own point